### ISSUE TITLE
schemdraw now uses a mpl figure wrapper, so drawCircuit is broken

### ISSUE THREAD

**Issue by lfry512** [Paul-Warburton-Research-Group/pycqed#2]
schemdraw has changed drastically: now it uses its own Figure class that wraps matplotlib functionality.

A current workaround in a Jupyter notebook is to just explicitly call the draw function on the underlying schemdraw drawing object in a cell:
```
circuit = CircuitSpec('My Circuit')
...
circuit.drawCircuit(inline=True)
circuit.drawing.draw()
```

**Comment by lfry512** [comment]
This was discovered after fix [6151e2e](https://github.com/Paul-Warburton-Research-Group/pycqed/commit/6151e2e5d71428133dfa0e1cb9dc199baa1442d1)

**Comment by lfry512** [comment]
No longer an issue as all circuit drawing has been removed for the time being.
